README.md

```diff
@@ -2,20 +2,24 @@
 
 <img src="https://github.com/bemanproject/beman/blob/main/images/logos/beman_logo-beman_library_under_development.png" style="width:5%; height:auto;">
 
+![Library Status](https://raw.githubusercontent.com/bemanproject/beman/refs/heads/main/images/badges/beman_badge-beman_library_under_development.svg)
+![Target Standard](https://github.com/bemanproject/beman/blob/main/images/badges/cpp29.svg)
+
+
 `beman.net` provides senders for asynchronous network operations.
 It is based on [P2762R2](https://wg21.link/P2762R2). Both the proposal
 and the implementation are in an early state although basic
 functionality is provided.
 
-`beman.net` builds on top of [`beman.execution26`](https://github.com/bemanproject/execution26). The
+`beman.net` builds on top of [`beman.execution`](https://github.com/bemanproject/execution). The
 implementation should allow multiple ways how to support asynchronous
 operations although currently the only implemented approach is based
 on `poll(2)`. Implementations using `kqueue(2)`, `epoll(2)`,
 `io_uring`, and IOCP hopefully follow soon. It should also be
 possible use various libraries for asynchronous operations like
 `libevent`, `libuv`, etc.
 
-**Implements**: [Sender/Receiver Interface for Networking (P2762R2)](https://wg21.link/P2762R2)
+**Implements**: [Sender/Receiver Interface for Networking (P2762)](https://wg21.link/P2762)
```

Comment:
let me know if you don't want this to float and I'll revert -- but seems like it's likely to track closely so less updates needed?